README.md

```diff
@@ -74,94 +74,7 @@ karpenter-5f97c944df-bm85s 1/1   Running 0        15m
 karpenter-5f97c944df-xr9jf 1/1   Running 0        15m
 ```
 
-You can now proceed to deploy the default Karpenter provisioner, and deploy any blueprint you want to test.
-
-#### Deploy a Karpenter Default Provisioner
```

Comment:
Let's rename this section to "## Karpenter Default Provisioner" and move it outside of this block, right before [Deploying a Blueprint](https://github.com/aws-samples/karpenter-blueprints#deploying-a-blueprint) as we're referencing this in several blueprints. We still need to explain this for those who are working with an existing cluster and didn't create the cluster using the EKS Blueprints.